### PR TITLE
UInt support

### PR DESCRIPTION
**Pull request by garbervetsky**
Addresses Issue #6.

Add assume (t>=0) to translated sub expressions. . May lead  to some code bloat.

### CODE REVIEW

**Comment by garbervetsky** [comment]
Please do not accept this PR yet because is breaking some regressions.

**Comment by garbervetsky** [comment]
> Please do not accept this PR yet because is breaking some regressions.

Now, all regressions pass.

File: Test/regressions/UInt.sol [hunk 1]
```diff
@@ -0,0 +1,44 @@
+pragma solidity >=0.4.24 <0.6.0;
+
+
+// Test uint implementation
+contract UintTest {
+  uint x;
+  uint256 x256;
```
Comment (shuvendu-lahiri) [inline Test/regressions/UInt.sol]: Can we add a couple of variables of type uint[], mapping (int => uint) and possibly uint[][] to see if we handle nested uints well?

Comment (shuvendu-lahiri) [reply Test/regressions/UInt.sol]: And possibly when passing a uint as a call argument?

Comment (garbervetsky) [reply Test/regressions/UInt.sol]: Done. I manually checked that all assertion actually failed when changing the conditions on variables. Should we include test with assertion failures?

**Pull request merged**